package.json

```diff
@@ -18,16 +18,20 @@
   },
   "homepage": "https://github.com/ng-bootstrap/core#readme",
   "devDependencies": {
-    "angular2": "^2.0.0-alpha.46",
+    "@reactivex/rxjs": "5.0.0-alpha.7",
```

Comment:
Yeah, but I thought it was only for the new version and not for the old ones.

I am waiting for angular .49 to fix an issue that prevents this library from upgrade and then updates rxjs too.